kits/ai-startup-idea-validator/apps/.env.example

```
@@ -0,0 +1,5 @@
+## Environment Variables
+
+```env
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The ```env key should be with a value or have an equal sign

(KeyWithoutValue)